webapp/components/sidebar_header.jsx

```diff
@@ -5,40 +5,66 @@ import React from 'react';
 
 import Client from 'client/web_client.jsx';
 import PreferenceStore from 'stores/preference_store.jsx';
+import UserStore from 'stores/user_store.jsx';
 import * as Utils from 'utils/utils.jsx';
 
 import SidebarHeaderDropdown from './sidebar_header_dropdown.jsx';
 import {Tooltip, OverlayTrigger} from 'react-bootstrap';
 
 import {Preferences, TutorialSteps, Constants} from 'utils/constants.jsx';
 import {createMenuTip} from 'components/tutorial/tutorial_tip.jsx';
+import StatusDropdown from './status_dropdown.jsx';
 
 export default class SidebarHeader extends React.Component {
     constructor(props) {
         super(props);
 
         this.toggleDropdown = this.toggleDropdown.bind(this);
         this.onPreferenceChange = this.onPreferenceChange.bind(this);
+        this.onStatusChange = this.onStatusChange.bind(this);
```

Comment:
Instead of having to explicitly bind these, you can actually change them to arrow functions when you define them to get rid of these lines. We've only started this in the last couple months, but it saves a lot of annoying binding. So you can remove this line and further down write:
```
onStatusChange = () => {
    ...
}
```

We don't need that for any of the React lifecycle functions (componentDidMount, render, etc), but it saves having to bind every listener and helper function to make `this` work